### DISCUSSION TITLE
upgrade from 1.17.0 to 2.0.1

### DISCUSSION THREAD

**Discussion by githubsharezone** [modoboa/modoboa#2528] [Help]
i have a modoboa installed on my raspberry pi 4

root@armlsrv:/opt/modoboa-2.0.1/modoboa-installer/modoboa_installer# cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
root@armlsrv:/opt/modoboa-2.0.1/modoboa-installer/modoboa_installer#

but whe i try execute upgrade with the follow link 

https://modoboa.readthedocs.io/en/1.6.0/getting_started/upgrade.html

and lunch 

root@armlsrv:/opt/modoboa-2.0.1/modoboa-installer/modoboa_installer# pip install modoboa==2.0.1
Collecting modoboa==2.0.1
  Could not find a version that satisfies the requirement modoboa==2.0.1 (from versions: 0.7.0, 1.2.0rc2, 1.2.0, 1.2.1, 1.2.2, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.3.5, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.4.5, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.6.0, 1.6.1, 1.6.2, 1.6.3, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.8.0, 1.8.1, 1.8.2, 1.8.3, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.10.3, 1.10.4, 1.10.5, 1.10.6, 1.10.7, 1.11.0, 1.11.1, 1.12.0, 1.12.1, 1.12.2, 1.13.0, 1.13.1, 1.14.0)
No matching distribution found for modoboa==2.0.1

can help me

**Comment by tonioo** [comment]
@githubsharezone Can you check the python3 version you use? Modoboa 2.0 requires python3.7 at least

**Reply by githubsharezone** [reply]
thnks for suggestion i execute the procedure and ok 

Successfully installed Pillow-9.1.1 PyYAML-6.0 aioredis-2.0.1 aiosmtplib-1.1.6 asgiref-3.5.2 async-timeout-4.0.2 attrs-21.4.0 bcrypt-3.2.2 certifi-2022.5.18.1 cffi-1.15.0 chardet-4.0.0 charset-normalizer-2.0.12 cryptography-37.0.2 deprecated-1.2.13 dj-database-url-0.5.0 django-3.2.13 django-ckeditor-6.3.0 django-filter-21.1 django-js-asset-2.0.0 django-otp-1.1.3 django-phonenumber-field-6.1.0 django-reversion-5.0.0 django-xforwardedfor-middleware-2.0 djangorestframework-3.13.1 djangorestframework-simplejwt-5.1.0 dnspython-2.2.1 drf-spectacular-0.22.1 feedparser-6.0.8 gevent-21.12.0 greenlet-1.1.2 idna-3.3 inflection-0.5.1 jsonfield-3.1.0 jsonschema-4.5.1 lxml-4.9.0 modoboa-2.0.1 oath-1.4.4 ovh-1.0.0 packaging-21.3 passlib-1.7.4 phonenumbers-8.12.49 progressbar33-2.4 py-dateutil-2.2 pycparser-2.21 pyjwt-2.4.0 pyparsing-3.0.9 pyrsistent-0.18.1 pytz-2022.1 qrcode-7.3.1 redis-4.3.1 requests-2.27.1 rrdtool-0.1.15 sgmllib3k-1.0.0 six-1.16.0 sqlparse-0.4.2 typing-extensions-4.2.0 uritemplate-4.1.1 urllib3-1.26.9 wrapt-1.14.1 zope.event-4.5.0 zope.interface-5.4.0
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv


but the modoboa not update see old release and old module
